File: csharp/MapleStory.Test/UserAPI.cs
using NUnit.Framework;
using MapleStory.OpenAPI.KMS;

namespace MapleStory.Test
{
    [TestFixture]
    public class UserAPI
    {
        private static readonly string apiKey = Environment.GetEnvironmentVariable("API_KEY")!; // Your API Key
        private static readonly MapleStoryAPI api = new MapleStoryAPI(apiKey);

        [Test]
        public async Task GetCharacterList()
        {
            var response = await api.GetCharacterList();
            Console.WriteLine(response.ToJson());
        }

        [Test]
        public async Task GetAchievement()
        {
            var response = await api.GetAchievement();
            Console.WriteLine(response.ToJson());
        }
    }
}

File: java/src/test/java/UserApi.java
import dev.spiralmoon.maplestory.api.kms.MapleStoryApi;
import dev.spiralmoon.maplestory.api.kms.dto.user.*;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import java.io.IOException;

@DisplayName("계정 정보 조회")
public class UserApi {

    private static final String apiKey = System.getProperty("API_KEY"); // Your API Key
    private static final MapleStoryApi api = new MapleStoryApi(apiKey);

    @Test
    @DisplayName("캐릭터 목록 조회")
    void getCharacterList() throws IOException {
        CharacterListDTO response = api.getCharacterList().join();
        System.out.println(response.toString());
    }

    @Test
    @DisplayName("업적 정보 조회")
    void getAchievement() throws IOException {
        AchievementDTO response = api.getAchievement().join();
        System.out.println(response.toString());
    }
}
